pkgs/development/python-modules/acstore/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  pyyaml,
+  setuptools,
+  ...
+}:
+buildPythonPackage rec {
+  pname = "acstore";
+  version = "20240407";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-yubHDEZ5nwltQW8sLEAhgyaXI0svHCS3a7Mewi6cvpg=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ pyyaml ];
+
+  disabled = pythonOlder "3.8";
+
+  pythonImportsCheck = [ pname ];
+
+  meta = with lib; {
+    changelog = "https://github.com/log2timeline/acstore/releases/tag/${version}";
+    description = "ACStore, or Attribute Container Storage, provides a stand-alone implementation to read and write attribute container storage files.";
```

Comment:
```suggestion
    description = "A stand-alone implementation to read and write attribute container storage files";
```
> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

Reply:
Will resolve and review all other packages for where I might have done this :+1: